src/liboslexec/llvm_util.cpp

```diff
@@ -8,11 +8,16 @@
 #include <OpenImageIO/thread.h>
 #include <boost/thread/tss.hpp>   /* for thread_specific_ptr */
 
+#ifdef OSL_USE_OPTIX
+#include <optix.h>
+#endif
+
 #include <OSL/oslconfig.h>
 #include <OSL/llvm_util.h>
 
 #if OSL_LLVM_VERSION < 70
 #error "LLVM minimum version required for OSL is 7.0"
+#include "oslexec_pvt.h"
```

Comment:
What does llvm_util need from oslexec_pvt.h?

In theory, llvm_util is standalone (-ish). ShadingSystem depends on llvmutil, not the other way around.

Reply:
That must have been leftover cruft/debugging-diagnostics.  Just updated the PR with these removed.